ruleReplaceAction.go

```diff
@@ -87,6 +88,12 @@ func (instance *ruleReplaceAction) contextRequestValueBy(name string) (string, b
 		return request.Method, true
 	case "host":
 		return request.Host, true
+	case "hostonly":
```

Comment:
Hi. Maybe you can change it to `hostName` and add another line for `hostPort`, too. That it might be aligned.

Reply:
How to get the default port in error case? Is it enough to test the scheme and return 80 or 443?

Reply:
Yes this makes sense.